app/src/form/support/FieldGroupHeader.js

```diff
@@ -7,9 +7,17 @@ const FieldGroupHeader = ({ index, name = '', description = '' }) => {
   return (
     <View>
       <View style={styles.fieldGroupHeader}>
-        <Text style={styles.fieldGroupName}>{!isNaN(index) ? `${index + 1}. ${name}` : name}</Text>
+        <Text style={styles.fieldGroupName} testID="text-name">
+          {!isNaN(index) ? `${index + 1}. ${name}` : name}
+        </Text>
+      </View>
+      <View style={styles.fieldGroupDescContainer}>
+        {description && (
```

Comment:
Use descriptive variable names that accurately represent the purpose of the variables.

```jsx
    {hasDescription && (
      <Text style={styles.fieldGroupDescription} testID="text-description">
        {descriptionText}
      </Text>
    )}
```

Reply:
noted. I'll fix it